pydeseq2/DeseqStats.py

```diff
@@ -202,6 +308,8 @@ def lfc_shrink(self):
                     offset,
                     prior_no_shrink_scale,
                     prior_scale,
+                    "L-BFGS-B",
```

Comment:
maybe add a comment that it's an optimizer

Reply:
I've added the argument name ("`optimizer`") in [3c0ded6](https://github.com/owkin/PyDESeq2/pull/32/commits/3c0ded613d38aacac433983c9871e3f3210c37b5)